app/controllers/settings_controller.rb

```diff
@@ -7,12 +7,12 @@ def index; end
   def deactivate_profile
     current_user.profile.inactive!
     sign_out current_user
-    redirect_to root_path, alert: t('deactivate_profile')
+    redirect_to root_path, alert: t('settings.deactivate_profile')
```

Comment:
Aqui vc poderia usar o padrão de mensagens para controller:
```suggestion
    redirect_to root_path, alert: t('.success)
```

E lá no settings.pt-BR.yml
```yml
t-BR:
  settings:
    deactivate_profile:
      success: Perfil desativado com sucesso
    delete_account:
      success: Conta excluída com sucesso
```

Reply:
Nos controllers é recomendável usar esse padrão, onde vc coloca o nome da action e embaixo o tipo da mensagem :)